app/javascript/src/components/Dashboard/Notes/constants.js

```diff
@@ -1,23 +1,22 @@
+import { buildSelectOption } from "utils";
 import * as yup from "yup";
 
-import { buildSelectOption } from "utils/index";
-
-export const ROLE_DATA = [
+export const ROLE_DATA = buildSelectOption([
   "UX Designer",
   "Tester",
   "Analyst",
   "Back-end developer",
   "Front-end developer",
-];
+]);
 
-export const TAGS_DATA = [
+export const TAGS_DATA = buildSelectOption([
```

Comment:
It should be plural. You are building options. Not a single option.

Reply:
Sure yedhin. I will fix this in the next PR.